src/components/DiscoverEnd/DiscoverEnd.jsx

```
@@ -1,4 +1,12 @@
-import { Box, Card, CardContent, IconButton, Typography } from "@mui/material";
+import {
+  Box,
+  Card,
+  CardContent,
+  IconButton,
```

ESLint

[error] 'IconButton' is defined but never used.

(no-unused-vars)